programs/conditional_vault/src/lib.rs

```diff
@@ -155,6 +155,104 @@ pub mod conditional_vault {
         Ok(())
     }
 
+    pub fn merge_conditional_tokens_for_underlying_tokens(
+        ctx: Context<MergeConditionalTokensForUnderlyingTokens>, amount: u64
+    ) -> Result<()> {
+        let accs = &ctx.accounts;
+
+        let vault = &accs.vault;
+        let vault_status = vault.status;
+        require!(vault_status == VaultStatus::Active, ErrorCode::VaultAlreadySettled);
+
+        // Store Pre-operation Balances
+        let pre_user_conditional_on_finalize_balance = ctx
+        .accounts
+        .user_conditional_on_finalize_token_account
+        .amount;
```

Comment:
weird indentation

Reply:
It should be auto formatted by cargo fmt but I haven't doubled check that it's doing this correctly for a little while.